src/providers/twitch/TwitchIrcServer.cpp

```diff
@@ -37,6 +39,15 @@ TwitchIrcServer::TwitchIrcServer()
 
     this->pubsub = new PubSub(TWITCH_PUBSUB_URL);
 
+    if (getSettings()->enableSevenTVEventApi)
+    {
+        this->eventApi = new SeventvEventApi(SEVENTV_EVENTAPI_URL);
+    }
+    else
+    {
+        this->eventApi = nullptr;
```

Comment:
The whole condition is not needed, just initialize the variable in the class/header to nullptr